tangelo/linq/target/backend.py

```diff
@@ -461,6 +461,9 @@ def _get_expectation_value_from_statevector(self, qubit_operator, state_prep_cir
             complex: The expectation value of this operator with regards to the
                 state preparation.
         """
+        if hasattr(self, "expectation_value_from_stabilizer_circuit") and initial_statevector is None:
```

Comment:
Could you please skip a line and leave a short comment about what case is handled here, for users ?